ostelco-ios-client/Storyboards/Login.storyboard

```diff
@@ -147,6 +148,14 @@
             </objects>
             <point key="canvasLocation" x="791" y="186"/>
         </scene>
+        <!--TabController-->
```

Comment:
why does this need an additional reference to the tab controller here?

Reply:
> This looks legit, but is there a particular reason not to wait until we consolidate this logic to one place?

yes, so we can make the awesome screen a part of the user testing next week

Reply:
> why does this need an additional reference to the tab controller here?

this is the entry point to the home screen